frontend/src/app/libs/user-profile-page/user-profile-page.component.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { UserFacadeService } from './store/user/user.facade';
+import { IUser } from './model/user';
+import { Router } from '@angular/router';
+import { EntityWrapper } from '../auth/models/entity-wraper';
+import { EntityStatus } from '../auth/models/entity-status';
+
+@Component({
+	selector: 'app-user-config',
+	templateUrl: './user-profile-page.component.html',
+	styleUrls: ['./user-profile-page.component.scss'],
+})
+export class UserProfilePageComponent implements OnInit, OnDestroy {
+	private destroySource: Subject<boolean> = new Subject<boolean>();
+	public profileForm: FormGroup;
+	public userForm: IUser;
+
+	constructor(private userFacadeService: UserFacadeService, private router: Router, private formBuilder: FormBuilder) {
+		this.profileForm = formBuilder.group({
+			email: [''],
+			password: [''],
+			mobilePhone: [''],
+			firstName: [''],
+			secondName: [''],
+		});
+	}
+
+	private navigateFromUserProfile(): void {
+		this.router.navigate(['/']);
+	}
+
+	private setValuesToForm(): void {
+		this.profileForm.setValue({
+			email: this.userForm.email,
+			password: this.userForm.password,
+			firstName: this.userForm.firstName,
+			secondName: this.userForm.secondName,
+			mobilePhone: this.userForm.mobilePhone,
+		});
+	}
+
+	private createChangedUser(value: any): any {
```

Comment:
not understand from first glance what this method does, so seems like we need to add more correct name

Reply:
done